docs/resources/lb_pool_v3.md

```diff
@@ -0,0 +1,85 @@
+---
+subcategory: "Dedicated Load Balance (Dedicated ELB)"
```

Comment:
suggest to move other elb v3 resources to this subcategory,
that will make the docs structure clear

Reply:
done